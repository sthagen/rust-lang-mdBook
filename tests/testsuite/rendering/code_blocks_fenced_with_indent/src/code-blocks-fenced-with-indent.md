# Code blocks fenced with indent

```rust
    // This has a first line that is indented.
    println!("hello");
```
