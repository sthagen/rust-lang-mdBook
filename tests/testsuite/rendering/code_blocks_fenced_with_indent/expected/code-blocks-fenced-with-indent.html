<h1 id="code-blocks-fenced-with-indent"><a class="header" href="#code-blocks-fenced-with-indent">Code blocks fenced with indent</a></h1>
<pre class="playground"><code class="language-rust"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>    // This has a first line that is indented.
    println!("hello");
<span class="boring">}</span></code></pre>